spec/support/corpus.rb

```diff
@@ -134,15 +138,33 @@ def install_mutant
         system(%w[bundle])
       end
 
-      # Not in the docs. Number from chatting with their support.
-      CIRCLE_CI_CONTAINER_PROCESSES = 2
+      # The effective ruby file paths
+      #
+      # @return [Array<Pathname>]
+      def effective_ruby_paths
+        paths = Pathname
+          .glob(repo_path.join('**/*.rb'))
+          .sort_by(&:size)
+          .reverse
+
+        paths - excluded_paths
+      end
+
+      # The excluded file paths
+      #
+      # @return [Array<Pathname>]
+      def excluded_paths
+        exclude.flat_map do |pattern|
+          Pathname.glob(repo_path.join(pattern))
+        end
+      end
 
       # Number of parallel processes to use
       #
       # @return [Fixnum]
       def parallel_processes
-        if ENV['CI']
-          Mutant::Config::DEFAULT.jobs
+        if ENV.key?('CI')
```

Comment:
I could, but `Mutant.ci?` should go away. Mutant itself should not be CI aware, for various reasons. It was left out in https://github.com/mbj/mutant/pull/502, as there is currently no direct option to steer the CLI progress formatter. I'll have to add a CLI option.

A spec suite is allowed to be CI aware, so I did not use this API. Maybe its time to add `MutantSpec.ci?` instead.